### DISCUSSION TITLE
OpenAI API key's free trial... and after that?

### DISCUSSION THREAD

**Discussion by georgepor** [nhaouari/obsidian-textgenerator-plugin#51] [Q&A] [answered]
To use the Text Generator plugin I have to create an account on OpenAI to get a free 18$ trial account.  

Do you know whether after the trial period expires, I should get their (expensive) commercial API key or I can still use the Text Generator plugin without it?

**Comment by nhaouari** [comment, marked as answer]
Thank you for your inquiry about OpenAI pricing and the Text Generator Plugin. 

OpenAI pricing is pay-as-you-go, and you can find more information at [https://openai.com/api/pricing/](https://openai.com/api/pricing/). 

![image](https://user-images.githubusercontent.com/9850722/206503050-7860645a-a7e2-4d48-a2da-6b3f395cc4ce.png)

Additionally, you may find the Text Generator Plugin helpful, which allows you to use other services such as huggingface with frontmatter configuration: [http://text-gen.com/frontmatter-configuration](http://text-gen.com/frontmatter-configuration).

Please let me know if there is anything else I can help with.

**Reply by georgepor** [reply]
Cool. Very affordable. Thank you.

**Reply by zzzSEA** [reply]
Can i use API with a proxy website? If so, how do i do it?
